src/electron/process_manager.ts

```diff
@@ -299,6 +301,13 @@ class ChildProcessHelper {
       }
```

Comment:
Could you add logging on exit with the exit code?

Reply:
Done in 58484896d9470a0bd4a271dc4ad70ef9e1e787ec.  We log the exit code if the processes shut itself down or the signal which caused it to terminate.